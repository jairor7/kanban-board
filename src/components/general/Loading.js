import React from "react";

const Loading = () => {
  return (
    <div className="spinner-content">
      <div className="spinner" />
      <span className="spinner-text">Cargando...</span>
    </div>
  );
};

export default Loading;
