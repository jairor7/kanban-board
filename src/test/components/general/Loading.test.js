import Loading from "../../../components/general/Loading";
import React from "react";
import toJSON from "enzyme-to-json";
import { shallow } from "enzyme";

test("should render Loading correctly", () => {
  const wrapper = shallow(<Loading />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
