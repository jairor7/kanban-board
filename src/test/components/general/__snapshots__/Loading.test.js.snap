// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Loading correctly 1`] = `
<div
  className="spinner-content"
>
  <div
    className="spinner"
  />
  <span
    className="spinner-text"
  >
    Cargando...
  </span>
</div>
`;
